docs/pages/getting-started/6-connecting-ehr.rst

```diff
@@ -0,0 +1,59 @@
+.. _connecting-crm:
+
+Getting Started on EHR integration
+##################################
+
+This getting started manual assumes the vendor and its clients (care organizations) are set up on the Nuts Network through :ref:`connecting-crm`.
+The next step is to integrate the vendor's electronic health record (EHR) with the Nuts node to execute Bolts.
+
+All APIs used in the following chapters are documented at the :ref:`API <nuts-node-api>` page.
+Open API Spec files are available for generating client code.
+
+Resolving Bolt endpoints
+************************
+
+Bolts define which technical endpoints should be defined for exchanging health information.
+These endpoints are grouped as services which are generally named after the Bolt they support.
+The Nuts registry (as described by :ref:`connecting-crm`) can be queried to find care organizations that support a particular Bolt,
+and to resolve the technical endpoints associated with it.
+
+Searching organizations
+=======================
+
+To find care organizations (registered in the Nuts registry) that support a specific Bolt, the search organization API can be used.
+It takes a query parameter that's used to match organization names and optionally a DID service type.
+If the DID service type is supplied the API only returns organizations which DID Document has a service with that type.
+
+.. code-block:: text
+
+    did:nuts:2mF6KT6eiSx5y2fwTP4Y42yMUh91zGVkbu4KMARvCJz9
+
+For example, the following API call searches the Nuts registry for organizations which name matches "Ziekenhuis" and have a service of type "eOverdracht-receiver" on their DID Document:
+
+.. code-block:: text
+
+    GET <internal-node-address>/internal/didman/v1/search/organizations?query=Ziekenhuis&didServiceType=eOverdracht-receiver
+
+.. note::
```

Comment:
just use an example service type, that won't break if we change the eOverdracht service type